scripting/ttt_show_infos.sma

```diff
@@ -21,23 +15,10 @@ new const g_szColors[][] =
   "#F57011"  // orange
 };
 
-public plugin_precache()
-{
-  static icon[32];
-  for(new i = 0; i <= charsmax(g_szIconNames); i++)
-  {
-    if(i < 5 && strlen(g_szIconNames[i]) < 3) continue;
-    formatex(icon, charsmax(icon), "gfx/ttt/%s.gif", g_szIconNames[i]);
-    precache_generic(icon);
```

Comment:
I think CS can't precache `.gif` images. If I use `.tga` format, I can't show it in MOTD, cause TGA is not supported by HTML 😢